codec/encoder/core/inc/param_svc.h

```diff
@@ -426,6 +432,11 @@ typedef struct TagWelsSvcCodingParam: SEncParamExt {
       pSpatialLayer->uiTransferCharacteristics = pCodingParam.sSpatialLayers[iIdxSpatial].uiTransferCharacteristics;
       pSpatialLayer->uiColorMatrix =             pCodingParam.sSpatialLayers[iIdxSpatial].uiColorMatrix;
 
+      pSpatialLayer->bAspectRatioPresent = pCodingParam.sSpatialLayers[iIdxSpatial].bAspectRatioPresent;
+      pSpatialLayer->eAspectRatio = pCodingParam.sSpatialLayers[iIdxSpatial].eAspectRatio;
+      pSpatialLayer->sAspectRatioExtWidth = BIG_ENDIAN_16(pCodingParam.sSpatialLayers[iIdxSpatial].sAspectRatioExtWidth);
```

Comment:
> #define WRITE_BE_32(ptr, val) this macro did this conversion. 

WRITE_BE_32(ptr, val) swaps bytes of raw data that were stored while assembling bits.

BIG_ENDIAN(x) macro swaps bytes of numerical value, that must be stored inside of this bits. The numerical value can arrive into write_bits procedure in little or big endianness. We know that this value is numerical, not just a 2 bytes of raw data, and this makes us know that the format of this numerical value must be big endian.

The fact that ffmpeg decoded correctly values, that were set with my broken macro (I used "ffprobe -show_streams | grep sample_aspect_ratio " with container that doesn't store sample aspect ratio) makes me think, that my check method was wrong..